tests/container_checker/test_container_checker.py

```diff
@@ -229,10 +229,15 @@ def test_container_checker(duthosts, enum_dut_feature_container, rand_selected_d
 
     pytest_require(container_name not in skip_containers,
                    "Container '{}' is skipped for testing.".format(container_name))
-    stop_container(duthost, container_name)
 
+    marker = loganalyzer.init()
     loganalyzer.expect_regex = get_expected_alerting_message(container_name)
-    with loganalyzer:
-        # Wait for 1 minutes such that Monit has a chance to write alerting message into syslog.
-        logger.info("Sleep 1 minutes to wait for the alerting message...")
-        time.sleep(70)
+
+    try:
+        loganalyzer.disable_log_rotate()
+        with loganalyzer:
+            stop_container(duthost, container_name)
+            time.sleep(30)
+            loganalyzer.wait_for_all_expected_events(marker, 180, 15)
```

Comment:
do we really need this?

can we do something like

with open(marker, ....) as loganalyzer:

Reply:
I am not sure what was the difference?

with loganalyzer is to make sure loganalyzer.analyze() is called in the end.

Reply:
Update as suggested.

Reply:
Remove log rotation calls.